test/fixtures/with-inline-plugins/.template-lintrc.js

```diff
@@ -1,9 +1,12 @@
-module.exports = {
+
+import inlineComponent from './rules/inline-component.js';
+
+export default {
   plugins: [
     {
       name: 'plugin1',
       rules: {
-        'inline-component': require('./rules/lint-inline-component')
```

Comment:
The `lint-` prefix on rule names is an artifact that was mostly removed years ago. I finished removing it on these fixture rules.